### PR TITLE
4.8 Changes

### PR DESCRIPTION
**Pull request by priyanka-TL**

### CODE REVIEW

**Comment by amorphous-1** [comment]
Hello @priyanka-TL,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://932.qa.docs.sunbird.org/4.8.0/ .<br/> **Note:** Only _4.8.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://932.qa.docs.sunbird.org/4.8.0/apis/mlcoreapi// <br /> - http://932.qa.docs.sunbird.org/4.8.0/apis/mlprojectsapi// <br />  <br/> **Modified sections are -** <br />

File: apis/mlcoreapi/mlcoreapi.yaml [hunk 1]
```diff
@@ -448,6 +448,12 @@ paths:
           schema:
             type: string
           example: 2515e6fbe5039a6db09c10148e3c743b
+        - name: createProject
```
Comment (anisha147) [inline apis/mlcoreapi/mlcoreapi.yaml:451]: 1. Line 419: Please change to "contain" in the sentence "The response will contains the below key." 
2. Line 420: Please check if this should come as a bullet point. 
3. Line 447: Lowercase "To" in the description
4. Global: Grammatically it should be "Represents" in the lines 53, 140, 297, 672, 958, 962, 966, 970, 1438, 1442, 1446, 1450, 1454, 1458, 1462, 1466

**Comment by amorphous-1** [comment]
Hello @priyanka-TL,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://932.qa.docs.sunbird.org/4.8.0/ .<br/> **Note:** Only _4.8.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://932.qa.docs.sunbird.org/4.8.0/apis/mlcoreapi// <br /> - http://932.qa.docs.sunbird.org/4.8.0/apis/mlprojectsapi// <br />  <br/> **Modified sections are -** <br />

**Comment by amorphous-1** [comment]
Hello @priyanka-TL,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://932.qa.docs.sunbird.org/4.8.0/ .<br/> **Note:** Only _4.8.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://932.qa.docs.sunbird.org/4.8.0/apis/mlcoreapi// <br /> - http://932.qa.docs.sunbird.org/4.8.0/apis/mlprojectsapi// <br />  <br/> **Modified sections are -** <br />

**Pull request merged**